app/presenters/tree_builder_alert_profile.rb

```diff
@@ -0,0 +1,59 @@
+class TreeBuilderAlertProfile < TreeBuilder
+  private
+
+  def tree_init_options(_tree_name)
+    {:full_ids => true}
+  end
+
+  def set_locals_for_render
+    locals = super
+    locals.merge!(
+      :id_prefix => "ap_",
+      :autoload  => true,
+    )
+  end
+
+  def alert_profile_kinds
+    MiqAlert.base_tables.sort_by { |a| ui_lookup(:model => a) }
+  end
+
+  # level 0 - root
+  def root_options
+    [N_("All Alert Profiles"), N_("All Alert Profiles")]
```

Comment:
` [t = N_("All Alert Profiles"), t]` ? so that we don't repeat calls? the same on other places too